src/MatrixClientPeg.js

```diff
@@ -147,7 +160,7 @@ class MatrixClientPeg {
         return matches[1];
     }
 
-    _createClient(creds: MatrixClientCreds) {
+    _createClient(creds: MatrixClientCreds, useIndexedDb) {
```

Comment:
Is `useIndexedDb` passed in somewhere? I don't see a change to any call site to `_createClient`.